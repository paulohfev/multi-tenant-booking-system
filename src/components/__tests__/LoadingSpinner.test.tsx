import { render } from '@testing-library/react'
import LoadingSpinner from '../LoadingSpinner'

describe('LoadingSpinner', () => {
  it('should match snapshot', () => {
    const { container } = render(<LoadingSpinner />)
    expect(container).toMatchSnapshot()
  })
})
