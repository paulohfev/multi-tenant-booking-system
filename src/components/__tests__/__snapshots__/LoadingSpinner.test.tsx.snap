// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner should match snapshot 1`] = `
<div>
  <div
    class="flex justify-center items-center h-full"
    role="status"
  >
    <div
      class="animate-spin rounded-full h-20 w-20 border-t-2 border-b-2 border-gray-900"
    />
  </div>
</div>
`;
